react/features/base/media/components/web/Audio.js

```diff
@@ -1,26 +1,29 @@
 /* @flow */
 
-import AbstractAudio from '../AbstractAudio';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AbstractAudio, { _mapDispatchToProps } from '../AbstractAudio';
 
 /**
  * The React/Web {@link Component} which is similar to and wraps around
  * {@code HTMLAudioElement} in order to facilitate cross-platform source code.
  */
-export default class Audio extends AbstractAudio {
-    /**
-     * {@code Audio} component's property types.
-     *
-     * @static
-     */
-    static propTypes = AbstractAudio.propTypes;
-
+class Audio extends AbstractAudio {
     /**
      * Implements React's {@link Component#render()}.
      *
      * @inheritdoc
      * @returns {ReactElement}
      */
     render() {
-        return super._render('audio');
+        return (
+            <audio
+                preload = 'auto'
+                ref = { this.setAudioElementImpl }
```

Comment:
I can try to add support for "canplay". But this way I will have to first cache the ref and then wait for the canplay event. Does that sound good ? On a side note I barely translated our current implementation which used to not care, but I agree that this would be a more correct approach.